shared/chat/inbox/row/big-team-rows.js

```diff
@@ -14,14 +14,14 @@ class BigTeamHeaderRow extends PureComponent<TeamProps> {
   render() {
     return (
       <HeaderBox>
-        <Avatar teamname={this.props.teamname} size={isMobile ? 24 : 16} />
+        <Avatar teamname={this.props.teamname} size={24} />
         <Text type="BodySmallSemibold" style={teamStyle}>{this.props.teamname}</Text>
         <Icon
           className="icon"
           type="iconfont-ellipsis"
           onClick={this.props.onShowMenu}
           style={{
-            fontSize: isMobile ? 20 : 16,
+            fontSize: 24,
```

Comment:
I think this will make the icon look too big, especially on desktop. Could we keep the current font sizes and just increase the padding around?

Reply:
I agree. I'm not quite used to working on a 4K monitor yet; on my laptop it does look too big.